charts/rancher-alerting-drivers/102.1.1/charts/prom2teams/templates/psp.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.global.cattle.psp.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
Aren't `PodSecurityPolicy` removed since K8S 1.25 ?
(ref. https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)

Reply:
Never mind. This chart already handles that when installed on K8S cluster that do not have that any longer.

```
#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
#{{- if .Values.global.cattle.psp.enabled }}
#{{- if not (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
#{{- fail "The target cluster does not have the PodSecurityPolicy API resource. Please disable PSPs in this chart before proceeding." -}}
#{{- end }}
#{{- end }}
#{{- end }}
```

(ref. charts/rancher-alerting-drivers/102.1.1/templates/validate-psp-install.yaml)